packages/@o3r/design/src/core/design-token/parsers/design-token.parser.ts

```diff
@@ -241,7 +240,7 @@ interface ParseDesignTokenFileOptions {
  * @param options
  */
 export const parseDesignTokenFile = async (specificationFilePath: string, options?: ParseDesignTokenFileOptions) => {
-  const readFile = options?.readFile || ((filePath: string) => fs.readFile(filePath, { encoding: 'utf8' }));
+  const readFile = options?.readFile || (async (filePath: string) => (await import('node:fs/promises')).readFile(filePath, { encoding: 'utf8' }));
```

Comment:
Why do we need to import it asynchronously?

Reply:
To be usable in non-node environment and remove the deps to it at file level.
Thanks to this change, we are able to replace these functions by some non-node based (and use the Design Token generation in a Chrome extension for example ;) )